Components/Debug/DebugTask.cpp

```diff
@@ -96,6 +100,12 @@ void DebugTask::HandleDebugMessage(const char* msg)
         // Echo the message (without the 'echo')
         CUBE_PRINT("\n%s", &msg[5]);
     }
+    else if (strncmp(msg, "readAccel ", 5) == 0) {
```

Comment:
The `strncmp` section is mainly for when there's something to be read after the first part of the command.

Here the third arg to strncmp is the length of bytes to compare, e.g. if you have "readAccel x" as a input, then it should be `10` (strlen("readAccel "))